pkgs/servers/tigerbeetle/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "tigerbeetle";
+  version = "unstable-2022-09-30";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetledb";
+    repo = pname;
+    rev = "2023-02-13-weekly";
+    sha256 = "sha256-OcVLFnzW140Drx/jQ6p3eW1ZDGXrRSM6eQ47L+Ub54A=";
```

Comment:
`sha256 = ` is an old way to specify hash attribute. Slightly here and more generic one is `hash = "sha256-OcVLFnzW140Drx/jQ6p3eW1ZDGXrRSM6eQ47L+Ub54A=";`.